README.md

```diff
@@ -2463,6 +2473,25 @@ This is a route-map for filtering networks sent to this area. Valid values are s
 ##### `filter_list_out`
 This is a route-map for filtering networks sent from this area. Valid values are string or keyword 'default'.
 
+##### `nssa`
+This property defines the area as NSSA (not so stubby area). Valid values are true, false or keyword 'default'. This property is mutually exclusive with `stub` and `stub_no_summary`.
+
+##### `nssa_default_originate`
+This property configures to originate Type-7 default LSA into NSSA area. Valid values are true, false or keyword 'default'.
+
+
+##### `nssa_no_redistribution`
+This property configures not to send redistributed LSAs into NSSA area. Valid values are true, false or keyword 'default'.
+
+##### `nssa_no_summary`
+This property configures not to send summary LSAs into NSSA area. Valid values are true, false or keyword 'default'.
+
+##### `nssa_route_map`
+This is the name of the Policy to control distribution of default route. Valid values are string or keyword 'default'. This MUST be set to 'default' when `nssa_default_originate` is 'default'.
+
+##### `nssa_translate_type7`
+This property translates LSA from type 7 to type 5. Valid values are 'always', 'always_supress_fa', 'supress_fa', 'never' or keyword 'default'.

```

Comment:
This needs a little more info:

```
Translates NSSA external (type 7) LSAs to standard external (type 5) LSAs for use outside the NSSA. Valid values are one of the following keyword strings:
```

Keyword | Description
:--|:--
always             | Always translate
suppress_fa       | Forwarding Address Suppression
always_suppress_fa | Always translate & use Forwarding Address Suppression
never               | Never translate
default             | Translation is not configured